marketplace.json

```diff
@@ -16,6 +16,9 @@
     },
     "CONFIG_PROJECT_NAME": {
       "label": "Config Project Name"
+    },
```

Comment:
Just wondering, was the block broken without this constant being specified?

Reply:
Are you referring to the new constant Domain Name? (I added that so we can use it in the link to Jira)